examples/nlp/bidirectional_lstm_imdb.py

```diff
@@ -41,12 +41,14 @@
 )
 print(len(x_train), "Training sequences")
 print(len(x_val), "Validation sequences")
+# Use pad_sequence to standardize sequence length:
```

Comment:
This file should not be included in this PR

Reply:
ok